src/app/containers/CpsFeaturesAnalysis/__snapshots__/index.test.jsx.snap

```diff
@@ -488,6 +488,7 @@ exports[`CpsRelatedContent should render Story Feature components when given app
         >
           <div
             class="c11"
+            data-e2e="story-promo"
```

Comment:
Do we strip these in production builds?

Reply:
I had thought about that, but unfortunately we can't, because it is used in the Cypress E2Es.